server.js

```diff
@@ -21,9 +21,73 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send("Hello!");
 });
 
+
+app.get("/bookshelf/page/:page", (req, res) => {
+  const page = req.params.page;
+  console.log(`requested page ${page}`);
+  const limit = 10
+  const startIndex = (page - 1) * limit;
+  const endIndex = page * limit;
+  const result = booksData.slice(startIndex, endIndex);
+  if (result) {
+    res.status(200).json({
+      success: true,
+      message: "Found books!",
+      body: {
+      books: result },
+  })
+}  else {
+  res.status(404).json({
```

Comment:
Nice way of solving the error messages - keep up the good work Edvard.
You're an inspiration for me :)